src/main/java/nl/novi/techiteasycontroller/Exceptions/IndexOutOfBounds.java

```diff
@@ -0,0 +1,4 @@
+package nl.novi.techiteasycontroller.Exceptions;
+
+public class IndexOutOfBounds {
```

Comment:
Dit is nog geen exception op deze manier. Je moet nog RuntimeExeption extenden en de super constructor(s) aanroepen.